Makefile

```diff
@@ -111,5 +111,12 @@ functest:
 	./hack/build/functests.sh
 
 functest-image-host: WHAT=tools/cdi-func-test-file-host-init
-functest-image-host:  manifests build
+functest-image-host:  manifests docker
 	${DO} ./hack/build/build-cdi-func-test-file-host.sh && ./hack/build/build-docker.sh "tools/cdi-func-test-file-host-init tools/cdi-func-test-file-host-http"
+
+cluster-sync-functest-image-init: WHAT = tools/cdi-func-test-file-host-init
+cluster-sync-functest-image-init: functest-image-host
```

Comment:
This isn't syncing anything to the cluster from what I can see.  The prerequisite should probably be `cluster-sync`.

Reply:
Yeah, I was a little optimistic here, I had run some prerequisite manually and didn't do a make clean to make sure everything got build properly. I will push an update soon that should be much better.